### PR TITLE
Fix GiSysFactors up/down

### PR DESCRIPTION
**Pull request by DenkMybu**
Fix for the issue implemented in the 15th of september PR concerning the up/down factors the Gi systematics

### CODE REVIEW

File: Analyzer/plugins/Analyzer.cc [hunk 1]
```diff
@@ -3980,7 +4031,8 @@ void Analyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
     }
     
     if (debug_ > 2 && trigInfo_ > 0) LogPrint(MOD) << "      >> Check if we pass Preselection";
-    bool passPre = passPreselection(passedCutsArray, true);
+    bool passPre = passPreselection(passedCutsArray, true);\
```
Comment (tvami) [inline Analyzer/plugins/Analyzer.cc:4034]:
```suggestion
    bool passPre = passPreselection(passedCutsArray, true);
```

File: Analyzer/plugins/Analyzer.cc [hunk 2]
```diff
@@ -6012,48 +5783,29 @@ void Analyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
...
-    tuple->PostS_ProbQNoL1VsFiStripsLogVsPt_Ias_down->Fill(1 - bestCandidateProbQNoL1, bestCandidateFiStripsLog*theGiSystFactorDown, bestCandidateTrack->pt(),  eventWeight_);
-    
-    // Systematics plots for PU rescaling
-    tuple->PostS_ProbQNoL1VsIas_Pileup_up->Fill(1 - bestCandidateProbQNoL1, bestCandidateIas,  eventWeight_ * PUSystFactor_[0]);
-    tuple->PostS_ProbQNoL1VsIas_Pileup_down->Fill(1 - bestCandidateProbQNoL1, bestCandidateIas,  eventWeight_ * PUSystFactor_[1]);
-    tuple->PostS_ProbQNoL1VsIasVsPt_Pileup_up->Fill(1 - bestCandidateProbQNoL1, bestCandidateIas, bestCandidateTrack->pt(), eventWeight_ * PUSystFactor_[0]);
-    tuple->PostS_ProbQNoL1VsIasVsPt_Pileup_down->Fill(1 - bestCandidateProbQNoL1, bestCandidateIas, bestCandidateTrack->pt(), eventWeight_ * PUSystFactor_[1]);
-    tuple->PostS_ProbQNoL1VsFiStripsVsPt_Pileup_up->Fill(1 - bestCandidateProbQNoL1, bestCandidateFiStrips, bestCandidateTrack->pt(), eventWeight_ * PUSystFactor_[0]);
-    tuple->PostS_ProbQNoL1VsFiStripsVsPt_Pileup_down->Fill(1 - bestCandidateProbQNoL1, bestCandidateFiStrips, bestCandidateTrack->pt(), eventWeight_ * PUSystFactor_[1]);
```
Comment (tvami) [inline Analyzer/plugins/Analyzer.cc:6048]: I dont think this block should be deleted

Comment (tvami) [reply Analyzer/plugins/Analyzer.cc:6048]: You are, this should be there only once

Comment (DenkMybu) [reply Analyzer/plugins/Analyzer.cc:6048]: So I'll keep those only within the loop "doSystsPlots_"

**Pull request merged**